compliance/collection/nodescanv2/nodescan.go

```diff
@@ -16,5 +19,60 @@ type NodeScan struct {
 
 // Scan scans the current node and returns the results as storage.NodeInventory object
 func (n *NodeScan) Scan(nodeName string) (*storage.NodeInventory, error) {
-	return nil, errors.New("Not implemented")
+	componentsHost, err := nodes.Analyze(nodeName, "/host/", false)
+	log.Info("Finished node inventory /host scan")
+	if err != nil {
+		log.Errorf("Error scanning node /host inventory: %v", err)
+	}
+	log.Infof("Components found under /host: %v", componentsHost)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
[nit] Did you mean this?
```suggestion
	if err != nil {
		log.Errorf("Error scanning node /host inventory: %v", err)
		return nil, err
	}
	log.Infof("Components found under /host: %v", componentsHost)
```